lib/commands/download-trades/handler.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const fs = require('fs')
+const _last = require('lodash/last')
+const { RESTv2 } = require('bfx-api-node-rest')
+const PromiseThrottle = require('promise-throttle')
+const chrono = require('chrono-node')
+const debug = require('../../debug')('>')
+const marketParam = require('../../params/market')
+const stringParam = require('../../params/string')
+
+const REQ_LIMIT = 1000
+const pt = new PromiseThrottle({
+  requestsPerSecond: 10.0 / 60.0,
+  promiseImplementation: Promise
+})
+
+/**
+  * Downloads the requested number of candles from a market identified by
+  * symbol/timeframe pair.
+  */
+module.exports = async (argv) => {
+  const rest = new RESTv2({ transform: true })
+  const output = stringParam('output', argv.output, true)
+  const market = marketParam('market', argv.market)
+  const to = stringParam('to', argv.to)
+  const toDate = chrono.parseDate(to)
+  const trades = []
+  const start = Date.now()
+  let lastTradeMTS = +toDate
+
+  if (!+isFinite(+toDate)) {
```

Comment:
`_isFinite` I see you used lodash but here native one.

Reply:
Nice catch, thanks